### DISCUSSION TITLE
AI幫你找蚊子

### DISCUSSION THREAD

**Discussion by Timmy110403540** [1101-GS3073-C/Python-Community-GS3073-C#192] [General]
美國的廠商Bzigo 於今年 CES 2022 線上展示間推出雷射蚊蟲追蹤裝置，利用AI來計算蚊子和灰塵的飛行軌跡，當有蚊子出現時會立即通知使用者。另外，它配備夜視功能的高解析攝影機，在晚上一樣也能偵測蚊子。但是這項科技目前還無法自動殺蚊，原因為目前對於雷射殺蟲的安全性有疑慮，所以無法自動殺除。這其實方便了許多，讓使用者不再需要大半夜跑起來找蚊子，找半天還找不到!

文章來源:https://technews.tw/2022/01/11/bzigo-infra-red-ai-mosquito-tracking-device-unveiled-in-ces-2022/

**Comment by cobb110202008** [comment]
如果這項技術能成功殺蚊，相信能提升夏天許多人的睡眠品質，包括我自己也是苦主之一。

**Comment by wh20502** [comment]
這個技術我認為遺憾的點是機器沒辦法判斷人眼球跟蚊子，如果人眼球虹膜出現在偵測範圍，也有可能被打，感覺影片中的小狗狗也有可能，畢竟高功率雷射不是一個很大眾化的安全商品。感覺實用性上還有一段距離要努力！

**Comment by ghost** [comment]
雖然不能殺蚊子 但是我覺得可以結合捕蚊燈 一發現有蚊子就自動打開捕蚊燈 也比較不會有安全的疑慮

**Comment by Shiba010** [comment]
感覺可以搭配能抓住蚊子的子彈使用！

**Comment by yasklop** [comment]
我覺的還是需要能殺蚊子的功能，畢竟大半夜爬起打來真的很累

**Comment by ayakasanma** [comment]
沒有殺蚊子的功能實在是🤦也只能偵測到停在牆上的，感覺不是最佳型態哈哈

**Comment by Lynn12378** [comment]
我覺得偵測蚊子可能不是我們需要的，打不打的到蚊子才是重點

**Comment by lx3211** [comment]
能夠殺死蚊子的殺傷力，如果不小心對準眼睛之類的部位，感覺還是有點小危險呢

**Comment by xCk27x** [comment]
我覺得光是能知道蚊子的大概位置就很不錯了
常常是知道有蚊子 但一開燈卻連個頭緒都沒有

**Comment by NcuJerryKao** [comment]
常常盯蚊子 人眼追蹤一陣子就消失了 找不到還要等蚊子現行滅殺才能睡覺
若有此AI幫忙追蹤的功能 那對普眾百姓有諸多好處

**Comment by Ben6uo** [comment]
但大部分的蚊子都沒害 這樣開新的東西可能會影響到大自然?

**Comment by skyling0299** [comment]
我同意說偵測蚊子跟打到蚊子是兩回事
如果可以在偵測的時候發送高強度光束，是否能把蚊子殺掉呢

**Comment by BingRong1217** [comment]
如果真的有效那可以提高很多人的睡眠品質

**Comment by limitx0** [comment]
能知道蚊子的位置是很有幫助
但有時會覺得要起來打死蚊子好麻煩

**Comment by Chenszuhong** [comment]
這個主題真是不錯的想法 感覺都很有趣而且很帥 想要知道更多能延生應用的地方

**Comment by awkward-willy** [comment]
台灣夏天蚊蟲真的很多，尤其蚊子還可能傳染登革熱等疾病，如果能及時檢測或許能解決許多問題。但我也認同應同時搭配誘捕等功能，否則即使檢測到可能也難以消滅。

**Comment by Raywei0214** [comment]
我覺得還可以應用到不只蚊子，可能蟑螂，老鼠，或是米蟲等等，但要對付這些蟲子用AI或許過為奢侈

**Comment by lukiea** [comment]
陶侃說過一句發人省思的話，大禹聖者，乃惜寸陰; 至於眾人，當惜分陰，豈能生無益於時，死無聞於後，是自棄也。希望大家能發現話中之話。儘管如此，我們仍然需要對蚊子保持懷疑的態度。帶著這些問題，我們一起來審視蚊子。經過上述討論，淮南子講過一句值得人反覆尋思的話，天下之物莫兇於鳩毒，然而良醫囊而藏之，有所用也。這段話可說是震撼了我。其實，若思緒夠清晰，那麼蚊子也就不那麼複雜了。領悟其中的道理也不是那麼的困難。一般來說，世界需要改革，需要對蚊子有新的認知。馬克思說過，在科學上沒有平坦的大道，只有不畏勞苦沿著陡峭山路攀登的人，才有希望達到光輝的頂點。這把視野帶到了全新的高度。

**Comment by larry40305** [comment]
結合雷射殺蚊子，夏天的晚上都會有雷射光秀

**Comment by 710388** [comment]
生活在現代實在是太好了，以後連蚊子都不用自己打了！！！ 再也不用蚊風而起

**Comment by ying0607** [comment]
看得到打不到🥲

**Comment by boson13579** [comment]
雷射蚊蟲追蹤裝置將AI用生活上，解決了很多人夏天睡眠的困擾。
